lib/template-loader.php

```diff
@@ -13,25 +13,7 @@
  * @return string[] List of all overrideable default template types.
  */
 function get_template_types() {
```

Comment:
Should we move this function to `default-template-types.php`? Seems similar to e.g. `gutenberg_get_indexed_default_template_types`